My suggestion of template

mokhin-denis opened the pull request:
Here I suggest a starter template on Bootstrap v3.3.5

mokhin-denis committed:
My first try simple-simple template with customized mod_menu, nod_search htnl views

mokhin-denis committed:
Update README.md

mokhin-denis commented:
Its not the end. need a big work to customize HTML views of Joomla modules to represent BS3

mbabker commented:
I'll go through this at some point in the next few days.  FWIW my idea if I ever found time was to build this out as a functional template using Bootstrap's [Starter Template](http://getbootstrap.com/examples/starter-template/) as a basis; nothing fancy but mostly to serve as a functionality demo of building a Bootstrap 3 based templates while making use of JHtml's API to cope with the needed changes from Bootstrap 2.

mokhin-denis commented:
And my suggestion here is to use protostar starter template and customize HTML views, and maybe create needed new modules (carousel, Jumbotron etc) and plug-ins for tabs, accordions, panels...

mbabker commented:
It isn't my aim to create a fully functional template with all the features of Bootstrap 3 represented.  Building a template based on Bootstrap's starter implies also building out layouts for all of the core extensions using Bootstrap 3 based markup, and using the plugin overrides fixing up the JHtml helpers for the Bootstrap JavaScript API.  But beyond that I didn't intend to add new features or extensions unless absolutely necessary to override some markup (I know the plugins are still pretty guilty of hardcoding HTML within them).

mokhin-denis committed:
Added correct layout for tags

mokhin-denis committed:
Added BS3 for 'calendar' field

Commit:
1. Added bootstrap.modal to plug-in for using BS3 Modal
2. Added new modChrome style to template
3. Added new HTML-template for mod_footer

mokhin-denis committed:
Added pagination. Fix menu. Up to BS3.3.6

mokhin-denis committed:
Added pagination

mokhin-denis committed:
Updated Bootstrap to latest 3.3.7

mokhin-denis committed:
Fixed an error with <> in module head. Fixed template

mokhin-denis committed:
Fixed row's markup and navigtion

mokhin-denis committed:
Fix conflict after merge

mokhin-denis committed:
Fixed error with Title tag for future support
